### ISSUE TITLE
Shifted side panel

### ISSUE THREAD

**Issue by Pixel48** [flameshot-org/flameshot#2343]
### Flameshot Version

Flameshot v11.0.0 (ad1bf28)
Compiled with Qt 5.15.2
winnt: 10.0.19043
windows: 10

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Windows 10 21H1

### Description

When system screen is expanded to 2nd monitor above, side panel is moved upwards

### Steps to reproduce

1. use windows
2. connect 2nd monitor and set it up to extend monitor upwards
3. run flameshot
4. hit space

### Screenshots or screen recordings

![obraz](https://user-images.githubusercontent.com/42846410/151696057-7e0094d7-9e34-4687-bdf8-07e039e26850.png)

### System Information

1. Windows 10 21H1
2. 
![obraz](https://user-images.githubusercontent.com/42846410/151683008-ca93072f-6861-412b-bc32-565a7483c9f6.png)

**Comment by mmahmoudian** [comment]
I tried with the following configuration and couldn't reproduce this on Manjaro Plasma X11

![image](https://user-images.githubusercontent.com/390889/151699521-00c3b492-9e1e-4c56-b539-f0a360d66502.png)

**Comment by Zackhardtoname** [comment]
In case it helps. On Linux, both of my screens would be shifted when taking screenshot unless I run `QT_SCREEN_SCALE_FACTORS='' flameshot gui` as the screenshot command.

**Comment by mmahmoudian** [comment]
@Zackhardtoname I think your issue is different as you most probably have fractional scaling on your monitors.

**Comment by Zackhardtoname** [comment]
> @Zackhardtoname I think your issue is different as you most probably have fractional scaling on your monitors.

My scaling is just 200% though. I am using KDE and i3.

**Comment by Pixel48** [comment]
I think i should mention: I have quite a difference in monitor size
![obraz](https://user-images.githubusercontent.com/42846410/152708182-c3667c05-3236-4851-969e-48e386a01aa5.png)

**Comment by mmahmoudian** [comment]
@Zackhardtoname you are right, 200% shod not give you any issues. Can you give us your monitors arrangement so that I try to replicate it.


@Pixel48 thank you for the info, but it was also visible in the screenshot you initially provided 😉 let's see if we can resolve this issue fast

**Comment by Zackhardtoname** [comment]
Hopefully, it would work.
```
Screen 0: minimum 8 x 8, current 5120 x 4320, maximum 32767 x 32767
DP-0 disconnected (normal left inverted right x axis y axis)
DP-1 disconnected (normal left inverted right x axis y axis)
HDMI-0 connected 3840x2160+731+0 (normal left inverted right x axis y axis) 597mm x 336mm
   3840x2160     60.00*+  59.94    50.00    30.00    29.97    25.00    23.98  
   1920x1200     59.88  
   1920x1080     60.00    59.94    50.00    23.98  
   1680x1050     59.95  
   1600x1200     60.00  
   1280x1024     75.02    60.02  
   1280x800      59.81  
   1280x720      59.94    50.00  
   1152x864      75.00  
   1024x768      75.03    60.00  
   800x600       75.00    60.32  
   720x576       50.00  
   720x480       59.94  
   640x480       75.00    59.94    59.93  
DP-2 disconnected (normal left inverted right x axis y axis)
DP-3 disconnected (normal left inverted right x axis y axis)
DP-4 disconnected (normal left inverted right x axis y axis)
DP-5 disconnected (normal left inverted right x axis y axis)
USB-C-0 connected primary 5120x2160+0+2160 (normal left inverted right x axis y axis) 800mm x 330mm
   3840x2160     60.00 +  59.94    50.00    29.97    25.00    23.98  
   5120x2160     60.00*   59.94    50.00    50.00    30.00    29.97  
   3440x1440     59.97    49.99    30.00  
   1920x1080     60.00    59.94    50.00    29.97    25.00    23.98  
   1680x1050     59.95  
   1600x1000     59.87  
   1280x1024     60.02  
   1280x960      60.00  
   1280x800      59.81  
   1280x720      60.00    59.94    50.00  
   1152x864      59.96  
   1024x768      60.00  
   800x600       60.32  
   720x480       59.94  
   640x480       59.94    59.93  
```

I have a similar setup as the OP. My monitors have the same scaling and PPI.
![image](https://user-images.githubusercontent.com/25935197/152745013-f1fa1b77-c91c-49dd-b18c-c4d08b6a0b31.png)

**Comment by mmahmoudian** [comment]
Ok then, confirmed. It is about monitor scaling in Linux and perhaps similar thing on Windows.

**Comment by jordi-sbd** [comment]
Hi guys, I have the same problem with 3 monitors (one of them is the laptop one):

![image](https://user-images.githubusercontent.com/98940443/219399686-47f9e143-13d4-4362-9b88-f1c5f0110d05.png)

and my windows 10 22H2 configuration is:

![image](https://user-images.githubusercontent.com/98940443/219400092-a9f2eea4-8c86-40f2-bfb4-6dccdd40ef70.png)

Regards,
Jordi

**Comment by panta82** [comment]
The fix proposed by @Zackhardtoname works for me

```
QT_SCREEN_SCALE_FACTORS='' flameshot gui 
```

**Comment by BenjaminSchuermann** [comment]
Hi,
i have the same Problem with 175% scaling and two monitors. One is UHD and the other on the right is FHD.
`QT_SCREEN_SCALE_FACTORS='' flameshot gui ` is solving the problem, but just for the one screenshot
if i use my shortcut, everything is shifted again

any workaround for this ?

**Comment by Zackhardtoname** [comment]
> Hi, i have the same Problem with 175% scaling and two monitors. One is UHD and the other on the right is FHD. `QT_SCREEN_SCALE_FACTORS='' flameshot gui ` is solving the problem, but just for the one screenshot if i use my shortcut, everything is shifted again
> 
> any workaround for this ?

Yeah I removed the shortcut from flameshot and set a system-wide shortcut to run that command

**Comment by metalredstar** [comment]
Hi... I experienced the same issue on Windows 10 (laptop 1366x768 primary display with an external secondary 1920x1080 display) but managed to find out the following:

With this screen positioning:
![image](https://github.com/flameshot-org/flameshot/assets/6917384/0384d541-77b8-458f-8ade-4117cf9f2082)

I get this issue:
![image](https://github.com/flameshot-org/flameshot/assets/6917384/b4fe4034-bd65-4d52-8512-9c1a87f5bfdb)


With this other screen positioning:
![image](https://github.com/flameshot-org/flameshot/assets/6917384/43e81c0a-82e7-4ad9-a7ca-e403fd548b3f)


everything is ok
![image](https://github.com/flameshot-org/flameshot/assets/6917384/35a38934-d211-4d37-9775-27c77def2268)


It seems like the overall screen area managed by Flameshot is the sum of the width of both screens the height is the height of the 
 wider display (I think it's because they are extended and positioned horizontally). So when you open the side panel it opens on the extreme left (the 0 value for X dimension which max value is Screen1X+Screen2X), BUT the panel is maximized on the Y dimension so it  gets the heigth of the wider display which Flameshot consider de height of it's working area.
 
 You should implement a way to manage irregular areas when extends different sized displays.

Hope this helps

**Comment by AhmeDseses** [comment]
Salam Alaykum

I found a temporary solution for this.

If it really annoys you and want to fix it, you can just get used to your new screen positions.

The attached image clears the idea.

<img width="797" alt="SS_2024-04-08_05_57_02" src="https://github.com/flameshot-org/flameshot/assets/98209948/d6216a47-5aa1-44d6-9240-44a52a570c8d">

**Comment by DZHEX** [comment]
I have a similar issue.
When the configuration is: 
![image](https://github.com/flameshot-org/flameshot/assets/20207609/993eaa00-850c-4e32-af98-77d595f578a9)
the options are off the screen (controls are "above" the monitor), like in the images above, but if it's changed to:
![image](https://github.com/flameshot-org/flameshot/assets/20207609/bd2cc716-b4a9-48d0-8e9a-25197564f24c)
then it's showing up fine.

It's somewhat annoying, that I have the screen I want to shoot on the right monitor (1), but the sidebar is showing up on the far left of the left monitor (2) (in both configurations), which is about a 90..150-degree turn for my head, depending on my pose xD
I think the sidebar should stick within the same monitor for less... head-turning, and should select left or right side of the monitor to avoid interfering with the content as much as possible.

That display 3 is some nvidia artifact caused by the VR headset, so I currently *can* move it wherever I want, since it is not displaying anything, but I do actually have a screen (a TV) that I might want to sometimes display stuff on above the center monitor, so the workaround of "just moving the displays" is a bit of a tough barrier to get over.

...As a side note - I'm also baffled that I require the sidebar to get to the color picker (pippette/dropper) tool... it would make sense as a button next to the actual screenshot area with all the rest of the tools. If I didn't need that sidebar to get to it, I wouldn't have noticed this issue (or the sidebar) whatsoever.
